readthedocs/subscriptions/utils.py

```diff
@@ -1,53 +1,70 @@
 """Utilities to interact with subscriptions and stripe."""
 
-import structlog
-
 import stripe
+import structlog
+from djstripe import models as djstripe
 from stripe.error import InvalidRequestError
 
 log = structlog.get_logger(__name__)
 
 
 def create_stripe_customer(organization):
     """Create a stripe customer for organization."""
-    stripe_customer = stripe.Customer.create(
+    stripe_data = stripe.Customer.create(
         email=organization.email,
         description=organization.name,
+        metadata=organization.get_stripe_metadata(),
     )
+    stripe_customer = djstripe.Customer.sync_from_stripe_data(stripe_data)
     if organization.stripe_id:
         log.warning(
-            'Overriding existing stripe customer. ',
+            "Overriding existing stripe customer.",
             organization_slug=organization.slug,
             previous_stripe_customer=organization.stripe_id,
             stripe_customer=stripe_customer.id,
         )
-    organization.stripe_id = stripe_customer.id
```

Comment:
We are still saving this data right, just it happens in the `save()` method?

Reply:
yeah, this is done in the save method now.